subsys/matter/Kconfig

```
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-# Support for Matter (formerly Connected Home over IP) protocol in NCS is experimental
+# Support for Matter (formerly Connected Home over IP) protocol in NCS is experimental for Wi-Fi technology
 config CHIP
 	bool
-	select EXPERIMENTAL
+	select EXPERIMENTAL if CHIP_WIFI
```

```suggestion
	select EXPERIMENTAL if WIFI_NRF700X
```
`CHIP_WIFI` is defined in `Kconfig.features` and may not be used by Matter users if they don't explicitly include it.